scripts/zones/Arrapago_Reef/mobs/Lil_Apkallu.lua

```diff
@@ -0,0 +1,16 @@
+-----------------------------------
+-- Area: Arrapago Reef
+--  ZNM: Lil Apkallu
+-----------------------------------
+mixins = {require("scripts/mixins/rage")}
+require("scripts/globals/status")
+-----------------------------------
+-- Todo: Apkallu hate, Hundred Fists, Movement and TP pattern
+
+function onMobInitialize(mob)
+    mob:setMobMod(dsp.mobMod.IDLE_DESPAWN, 300)
+    mob:setLocalVar("[rage]timer", 3600) -- 60 minutes
```

Comment:
this one too

Reply:
localvars should never be set in the init.